### ISSUE TITLE
Wrong XML Doc comment for type providers static parameters

### ISSUE THREAD

**Issue by vasily-kirichenko** [tpetricek/FSharp.Formatting#434]
![image](https://cloud.githubusercontent.com/assets/873919/24972235/61a5ae88-1fc3-11e7-9892-0606939a6919.png)

**Comment by matthid** [comment]
@vasily-kirichenko To be honest I'm not sure what we (FSharp.Formatting) did wrong here. Maybe I'm blind, sorry :)

**Comment by wallymathieu** [comment]
Isn't YamlConfig defined in https://github.com/fsprojects/FSharp.Configuration ?

**Comment by matthid** [comment]
Yes, but still I don't get it. Maybe it would help to tell what you expect to see in the doc comment?
Basically we just show what people enter into the doc-comments. For type-providers it seems to look like this:

https://github.com/fsprojects/FSharp.Configuration/blob/0900284a739ec54f1397b9e229df376f45fd4cb3/src/FSharp.Configuration/YamlConfigProvider.fs#L517-L521

Which seems to be exactly the content of the popup..
